Ifeanyi/Chapter_1/exercise_1.12.5

```diff
@@ -0,0 +1,8 @@
+Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
+[GCC 5.4.0 20160609] on linux
+Type "copyright", "credits" or "license()" for more information.
+>>> cheese
+Traceback (most recent call last):
+  File "<pyshell#0>", line 1, in <module>
+    cheese
+NameError: name 'cheese' is not defined
```

Comment:
The focus of this exercise is a "NameError" in python. We know it happens at run-time but there is a name for such an error in python. So update your commit message and PR message.